apps/src/code-studio/components/SoundLibrary.jsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import SoundList from './SoundList';
+import SoundCategory from './SoundCategory';
+import * as color from "../../util/color";
+
+const SOUND_CATEGORIES = {
+  category_digital: 'Digital',
+  category_gameplay: 'Game play',
+  category_jingles: 'Jingles',
+  category_female_voiceover: 'Female voiceovers',
+  category_male_voiceover: 'Male voiceovers',
+  category_objects: 'Objects',
+  category_ui: 'User interfaces',
+  category_all: 'All'
+};
+
+const styles = {
+  searchArea: {
+    float: 'right',
+    position: 'relative',
+    margin: '10px 0'
+  },
+  input: {
+    width: 300,
+    border: '1px solid #999',
+    borderRadius: 4,
+    padding: '3px 7px'
+  },
+  sound: {
+    position: 'absolute',
+    right: 5,
+    top: 5,
+    fontSize: 16,
+    color: '#999'
```

Comment:
Ideally we would not be adding new colors, but using something from color.scss. If we do need a new color here, we should add it to color scss and use it here (line 25 as well).